### DISCUSSION TITLE
在重啟模擬器功能上標示未完成

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#2989] [Ideas]
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/29662367/6da719c3-52a6-4c2b-a369-59a073c63017)

避免重複詢問，重複的issue滿多的
~~雖然能理解有這需求但lme還沒弄完，各位再等等~~

**Comment by LostMyMind** [comment]
感觉这个功能鸽了挺久了，我记得去年年末就重写了，现在写了快一年了
